src/UI/Factory.php

```diff
@@ -336,4 +337,62 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers can be used if space is scarce to display secondary information such as a preview of
+	 *     some item or to offer limited user interaction.
```

Comment:
How does this cover the background task or the On-screen-Chat at all? It is not a preview. It certainly is not secondary information to an object. 

Suggestion: Popovers can be used when space is scarce i.e. within List GUI items, table cells or  menus in the Header section. They offer either secondary information on object like a preview or rating to be displayed or entered. They display information about ongoing processes